lib/about_yml/schema.json

```diff
@@ -4,11 +4,11 @@
     "$schema": "http://json-schema.org/draft-04/hyper-schema",
     "type": "object",
     "properties": {
-        "name": {
+        "id": {
```

Comment:
The change from "name" => "id" and "full_name" => "name" is significant enough I just want to draw people's attention to it. When this change is integrated into this gem, we'll have to make sure to switch the Team API and Dashboard over to it, and then flip the switch. I'll take a task to set up a staging instance of the Team API for this purpose.